sdks/java/core/src/test/java/org/apache/beam/sdk/schemas/SchemaCoderTest.java

```diff
@@ -290,5 +329,31 @@ public void coderConsistentWithEquals() throws Exception {
         }
       }
     }
+
+    @Test
+    public void verifyDeterministic() throws Exception {
+      if (expectDeterministic) {
+        assertDeterministic(coder);
+      } else {
+        assertNonDeterministic(coder);
+      }
+    }
+  }
+
+  private static void assertDeterministic(SchemaCoder coder) {
```

Comment:
There's also https://github.com/apache/beam/blob/0098eb6f9900849b05bfc2ec2497ed8c7d37148a/sdks/java/core/src/main/java/org/apache/beam/sdk/testing/CoderProperties.java#L67

(coder properties are "properties" in the sense of property-driven testing a la QuickCheck)

Reply:
Thanks! Added a commit to use verifyDeterministic.